laravel/app/Http/Controllers/SlotController.php

```diff
@@ -188,4 +188,27 @@ public function adminRelease(Request $request, Slot $slot)
             return redirect('/event/' . $slot->event->id);
         }
     }
+
+    public function adminAssign(Request $request, Slot $slot)
+    {
+        $input = (int)$request['user-report'][0];
+        $users = User::get()->all();
+
+        if(is_null($slot->user) && Auth::user()->hasRole('admin'))
+        {
+            foreach ($users as $user) 
+            {
+                if($user->data->user_id==$input)
+                {
+                    $slot->user_id=$user->data->user_id;
+                    $username = (!is_null($user->data->burner_name) ? $user->data->burner_name : $user->data->full_name);
```

Comment:
This should use the same logic as the report controller instead of using `!is_null()`